src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import styles from './index.module.css';
+import menuIcon from '~/assets/img/icons/menu-icon.svg';
+import closeIcon from '~/assets/img/icons/close-icon.svg';
+import { Menu } from './config';
+import { ListLine } from '../common/ListLine';
+import { ListSocialNetwork } from '~/utils/const';
+
+const Sidebar = () => {
+  const [menuOpen, setMenuOpen] = useState(false);
+  console.log(menuOpen);
+
+  const toggleMenu = () => {
+    setMenuOpen(!menuOpen);
+  };
+
+  const sidebarStyle = {
+    left: menuOpen ? '0' : '-260px',
+  };
+
+  return (
+    <div className={styles.sidebar} style={sidebarStyle}>
+      <button className={styles.btnMenu} onClick={() => toggleMenu()}>
```

Comment:
```suggestion
      <button className={styles.btnMenu} onClick={toggleMenu}>
```

try to avoid inline functions inside components. performance issues

https://dev.to/karthick30/common-react-performance-mistakes-ac4